frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/EditStepBlock.tsx

```diff
@@ -57,6 +57,22 @@ export const EditStepBlock = ({
   }, [])
 
   const handleSubmit = formMethods.handleSubmit((inputs: EditStepInputs) => {
+    if (isFirstStepByStepNumber(stepNumber)) {
+      if (inputs.field) {
+        return onSubmit({
+          ...inputs,
+          workflow_type: WorkflowType.Dynamic,
+          field: inputs.field,
+        })
+      } else {
```

Comment:
/nit don't need an `else`, since we're returning in the control statement

Reply:
Resolved!